src/Sulu/Bundle/AdminBundle/Resources/js/index.js

```diff
@@ -389,6 +389,19 @@ function startAdmin() {
         'font-family: monospace; font-size: 10px; text-decoration: none;'
     );
 
+    const id = 'application';
+    const applicationElement = document.getElementById(id);
+
+    if (!applicationElement) {
+        throw new Error(`DOM element with ID "${id}" was not found!`);
+    }
+
+    if (!('config' in applicationElement.dataset)) {
+        throw new Error('Configuration not found');
+    }
+
+    Object.assign(Config, JSON.parse(applicationElement.dataset.config));
```

Comment:
Looks like we freezed the Config object in the main.html.twig currently. We should do the same currrently here.

```suggestion
    Object.assign(Config, JSON.parse(applicationElement.dataset.config));
    Object.freeze(Config);
```